CODEOWNERS

```diff
@@ -14,4 +14,5 @@
 # In each subsection folders are ordered first by depth, then alphabetically.
 # This should make it easy to add new rules without breaking existing ones.
 
+packages/admin-ui-components @dhartunian @nkodali @nathanstilwell
 *   @nathanstilwell @j-low @laurenbarker
```

Comment:
```suggestion
*   @nathanstilwell @j-low @laurenbarker
packages/admin-ui-components @dhartunian @nkodali @nathanstilwell
```

Reply:
ah thanks for actually reading the docs!